.github/CODEOWNERS

```diff
@@ -1 +1,12 @@
 
+
+/heresanother.txt @meichend
```

Comment:
I'm commenting on the line "/heresanother.txt @meichend"

Reply:
wow

Reply:
@derrickpersson this change seems wrong...